test/langtools/jdk/javadoc/doclet/testPreview/TestPreview.java

```diff
@@ -156,6 +157,9 @@ public void testPreviewAPIJavadoc() {
                     <li><a href="package-summary.html">preview</a></li>
                     <li><a href="Core.html" class="current-selection">Core</a></li>
                     </ol>""");
+
+        // 8331947: Support preview features without JEP should not be included in Preview API page
```

Comment:
Trivial: here it is phrased as "Support preview features", while in the actual test source file it is "Preview support feature".

Reply:
I agree the terminology in comments is somewhat inconsistent. I don't think there is an "official" term for these preview features. [In the CSR](https://bugs.openjdk.org/browse/JDK-8329634) they are summarized as follows: "Add a set of persistent preview visitors to support new language changes in development." 

I think the essential property for our purpose is the "without JEP". I added the "support" adjective to make their purpose clearer, but I don't think it's essential.